datalad_crawler/pipelines/tests/test_openfmri.py

```diff
@@ -361,7 +365,9 @@ def hexsha(l):
     ok_file_under_git(opj(outd, 'changelog.txt'), annexed=False)
     ok_file_under_git(t1w_fpath, annexed=True)
 
-    from datalad.metadata.metadata import agginfo_relpath
+    from datalad.metadata.metadata import get_ds_aggregate_db_locations
+    ds = Dataset('.')
+    dbloc, objbase = get_ds_aggregate_db_locations(ds)
```

Comment:
I wonder it to make it life easier we could support old way if import fails? Then leave a note to remove it upon next release of datalad (and boost version dependency). Otherwise we would not be able to release crawler fixes until datalad release and curse versa

Reply:
Will do.

Reply:
Done in #33